.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Can this file be removed?

I guess we can add `.vscode` to `.gitignore` if you are using it and don't want to pollute repo by mistake.